application/lib/help.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+
+class HelpTab extends StatefulWidget {
+  const HelpTab({Key key}) : super(key: key);
+
+  @override
+  _HelpTabState createState() => _HelpTabState();
+}
+
+class _HelpTabState extends State<HelpTab>{
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        ListView(
+          children: const <Widget>[
+            Text(
+                "To create a TAtom from scratch, go to the 'editor' tab and press the plus button. "
```

Comment:
add headers

`Text('Introduction', style: Theme.of(context).textTheme.headline4),` or something